COLLABORATOR_GUIDE.md

```diff
@@ -263,6 +263,20 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Introducing New Modules
+
+Semver-minor commits that introduce new core modules should be treated with
+extra care.
+
+* First, the name of the new core module should not conflict with any existing
+  module in the ecosystem unless an agreement with the owner of those modules
+  is reached to transfer ownership.
+* PRs that introduce new core modules must be landed as semver-minor commits.
```

Comment:
I think this is unnecessary to state. Any new feature must be landed as semver-minor.  If you're trying to say that even landing as `Experimental` is semver-minor (which I agree it should be), put that in with the statement about it needing to be `Experimental` first.